knowledge-base/how-to-upgrade-an-old-project-to-the-latest-telerik-reporting-version.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to upgrade an old project to the latest Telerik Reporting version
+description: Upgrade your old project successfully and catch up with the API changes in the later versions. This is a guide to upgrade old Telerik Reporting project to the latest version.
+type: how-to
+page_title: Upgrade from old to latest version of Telerik Reporting
+slug: how-to-upgrade-an-old-project-to-the-latest-telerik-reporting-version
+res_type: kb
+category: knowledge-base
+---
+
+## DESCRIPTION
+You have an old Telerik Reporting project and you want to upgrade it to the latest version. When you upgrade an old project directly to the latest version of Telerik Reporting you may see build errors. The reason is API changes introduced during the years.  
+
+This article will guide you on the steps to upgrade to the latest version of Telerik Reporting without build errors.
+
+## SOLUTION   
+To upgrade Telerik Reporting to the latest version without a problem you can rely on the fact that we keep obsolete code for at least an year before changing the API. In the transition period Report Designers are configured to reserialize the obsolete code from the report's definition, and to update it with the new standard.  

```

Comment:
This can go in the description as it explains the situation and does not suggest a solution.

Reply:
moved the first paragraph in the description and removed "This article will guide you on the steps to upgrade to the latest version of Telerik Reporting without build errors." from there.